BlueprintUI/Sources/AttributedText/AttributedText.swift

```diff
@@ -0,0 +1,149 @@
+import UIKit
+
+/// `AttributedText` allows you to apply strongly-typed attributes to strings (much like the `AttributedString` type
+/// introduced in iOS 15). You can then access the `attributedString` property to get an attributed string with those
+/// attributes applied.
+///
+/// For example:
+///
+/// ```swift
+/// var text = AttributedText(string: "Hello, world")
+/// // Apply a font to the entire range
+/// text.font = .systemFont(ofSize: 20)
+///
+/// // Apply a color to part of the string
+/// let range = text.string.range(of: "world")!
+/// text[range].color = .blue
+///
+/// // Render the attributed text
+/// let label = AttributedLabel(attributedText: text.attributedString)
+/// ```
+///
+@dynamicMemberLookup public struct AttributedText {
+
+    /// The wrapped string, with no attributes.
+    public let string: String
+
+    /// An `NSAttributedString` representation of the attributed text.
+    public var attributedString: NSAttributedString {
+        mutableAttributedString
```

Comment:
Nit: Return a `.copy()` of this – otherwise you're returning the actual underlying mutable value, which can continue to mutate.